### DISCUSSION TITLE
Remote Administrator Error: No Channel

### DISCUSSION THREAD

**Discussion by chairmanfaust** [meshtastic/Discussions#105] [General]
I'm trying the new v2.5 remote admin method and I'm wondering if i still need a secondary channel named admin on all devices despite not using the 'legacy admin' switch. I have the admin key entered but get the 'no channel' error when i try to change any settings on a managed node. 

![Screenshot_20241203_082604_Meshtastic 1](https://github.com/user-attachments/assets/ce1c6d58-0073-4243-80a1-b8ed583ae1ca)

**Comment by garthvh** [comment]
It means the nodes have not exchanged key yet

**Reply by chairmanfaust** [reply]
How does that occur?  both nodes have been online for about 12 hours now with the client containing the admin key of the master node's public key

**Reply by garthvh** [reply]
You can see if a key exchange has taken place in the connected phone app via the green lock icon. Without a green lock you will get a no channel error.

**Reply by chairmanfaust** [reply]
green lock confirmed
![Screenshot_20241203_103734_Meshtastic 1](https://github.com/user-attachments/assets/fea0d0cf-47fb-4caa-ba9b-e0827309db7c)

**Comment by chairmanfaust** [comment]
The green checkbox appeared from the master node but on the managed node the master had a red key, so once i reset the nodedb it became green and started working as expected
